TowerForge/TowerForge/TFCore/TFLabelNode.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  TFLabelNode.swift
+//  TowerForge
+//
+//  Created by Zheng Ze on 29/3/24.
+//
+
+import SpriteKit
+
+enum AlignmentMode {
```

Comment:
Should we store this in a constants file?

Reply:
Hmmm, this is only used here currently so I feel that it should be here so that we can easily find it. But I have no issues with moving it as well if needed.